# ICRA状态

Header header

# Buff区状态
rm_referee_controller/ICRABuff buff
# 剩余子弹数量
rm_referee_controller/ICRARemainingBullet remainingBullet